### PR TITLE
8282936: Write a regression test for JDK-4615365

### PR DESCRIPTION
**Pull request by manukumarvs**
Write a regression test for [JDK-4615365](https://bugs.openjdk.java.net/browse/JDK-4615365) : JSplitPane current and last divider positions incorrect when realized

Issue:
JSplitPane component with a left and right component has its current divider
position incorrectly set to -1 once it is fully realized. For JDK 1.3.1, its
value is set to 0. The current position of the divider should be the current
position of the divider once layout management is complete. If the divider is
moved one pixel, the current divider position is updated to the correct value
but the last divider position is then set to -1 (i.e. the last value of the
current divider position). If the divider is moved again, everything is OK.

Testing:
Java 1.4.0 -> Test Failed.
$ j2sdk1.4.0/bin/java bug4615365
current= -1, last= 0
Test Failed.

Java 1.4.1 -> Test Passed.
$ j2sdk1.4.1/bin/java bug4615365
current= 97, last= -1
Test Passed.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8282936](https://bugs.openjdk.java.net/browse/JDK-8282936): Write a regression test for JDK-4615365

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7802/head:pull/7802` \
`$ git checkout pull/7802`

Update a local copy of the PR: \
`$ git checkout pull/7802` \
`$ git pull https://git.openjdk.java.net/jdk pull/7802/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7802`

View PR using the GUI difftool: \
`$ git pr show -t 7802`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7802.diff">https://git.openjdk.java.net/jdk/pull/7802.diff</a>

</details>

### CODE REVIEW

File: test/jdk/javax/swing/JSplitPane/4615365/JSplitPaneDividerLocationTest.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+import javax.swing.UIManager;
+import javax.swing.UIManager.LookAndFeelInfo;
+import javax.swing.UnsupportedLookAndFeelException;
+
+import static javax.swing.UIManager.getInstalledLookAndFeels;
+/*
+ * @test
+ * @key headful
+ * @bug 4615365
+ * @summary This test confirms that the JSplitPane current and last divider positions are correct when realized.
```
Comment (mrserb) [inline test/jdk/javax/swing/JSplitPane/4615365/JSplitPaneDividerLocationTest.java:52]: Please split the long lines to have 80 chars per line.

Comment (manukumarvs) [reply test/jdk/javax/swing/JSplitPane/4615365/JSplitPaneDividerLocationTest.java]: Done.

**Comment by manukumarvs** [comment]
@mrserb I have fixed the review comments, can you please have a look.

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**